Update scalardb getting started document (#93)
scalar-labs/scalardb: docs/getting-started.md

$ cd $SCALARDB_HOME
$ ./gradlew installDist

> $ sudo mkdir /var/log/scalar; sudo chmod 777 /var/log/scalar

Reviewer: ```suggestion
$ sudo mkdir /var/log/scalar;
$ sudo chmod 777 /var/log/scalar
```

Author: Thank you for your suggestion.
Document updated with the above suggestion.

$ cd $SCALARDB_HOME
$ ./gradlew installDist
$ sudo mkdir /var/log/scalar; sudo chmod 777 /var/log/scalar
$ cd tools/scalar-schema

> $ lein uberjar

Reviewer: I don't think it is user-friendly to make users build the jar in the getting started document.
Since it is jar, we should upload the jar to GitHub and just use it.
Thought?

Author: Thank you for the review comments.
jar file added into schema-tool and the document updated accordingly.

First of all, you need to define how the data will be organized (a.k.a database schema) in the application with Scalar DB database schema.
Here is a database schema for the sample application. For the supported data types, please see [this doc](schema.md) for more details.

```json
{
  "emoney.account": {
    "transaction": false,
    "partition-key": [
      "id"
    ],
    "clustering-key": [],
    "columns": {
      "id": "TEXT",
      "balance": "INT"
    },

>     "compaction-strategy": "STCS"

Reviewer 2: I think this line can be removed since `compaction-strategy` is optional and the default value is "STCS".
The compaction strategy isn't important in this document.

Author: Thank you for your suggestion.
Document updated with the above suggestion.

  }
}
```

To load the schema file, please run the following command.
```

> $ java -jar $SCALARDB_HOME/tools/scalar-schema/scalar-schema.jar --cassandra -h localhost -u <CASSNDRA_USER> -p <CASSANDRA_PASSWORD> -f emoney-storage.json -R 1

Reviewer: Please add the jar to the releases and use it. (so we need a download step)
I think each release should have the jar built with the release.

Author: Thank you for your suggestion.
Document updated with the above suggestion.

scalar-schema-2.2.1.jar available in [scalardb release v2.2.1](https://github.com/scalar-labs/scalardb/releases/tag/v2.2.1)

`scalar-schema.jar` updated with scalardb release version number. `v` is not using with jar file because the source (zip) file does not contain `v`,  it downloads as `scalardb-2.2.1.zip`

First of all, you need to define how the data will be organized (a.k.a database schema) in the application with Scalar DB database schema.
Here is a database schema for the sample application. For the supported data types, please see [this doc](schema.md) for more details.

```json
{
  "emoney.account": {
    "transaction": false,
    "partition-key": [
      "id"
    ],
    "clustering-key": [],
    "columns": {
      "id": "TEXT",
      "balance": "INT"
    }
  }
}
```

> To load the schema file, please run the following command.

Reviewer: ```suggestion
Then, download the schema loader that matches with the version you use from [scalardb releases](https://github.com/scalar-labs/scalardb/releases), and run the following command to load the schema.
```

Author: Thank you for the suggestion.
Document updated with the above suggestion.

> * Download `scalar-schema-<vesrion>.jar` file from [scalardb release](https://github.com/scalar-labs/scalardb/releases)

Reviewer: ```suggestion
```

Reviewer: It's weird to see this with a bullet point after saying `run the following`. Please explain it properly.

Author: Thank you for the suggestion.
Document updated with the above suggestion.

$ ../../gradlew run --args="-mode storage -action pay -amount 100 -to merchant1 -from user1"
```

## Set up database schema with transaction

To apply transaction, we can just add a key `transaction` and value as `true` in Scalar DB scheme.

```json
{
  "emoney.account": {
    "transaction": true,
    "partition-key": [
      "id"
    ],
    "clustering-key": [],
    "columns": {
      "id": "TEXT",
      "balance": "INT"
    },

>     "compaction-strategy": "STCS"

Reviewer 2: ditto

Author: Thank you for your suggestion.
Document updated with the above suggestion.

  }
}
```

Before reapplying the schema, please drop the existing namespace first by issuing the following. 

> (Sorry you need to issue implementation specific commands to do this.)

Reviewer: ```suggestion
```

Author: Thank you for your suggestion.
Document updated with the above suggestion.

```

> $ java -jar $SCALARDB_HOME/tools/scalar-schema/scalar-schema.jar --cassandra -h localhost -u <CASSNDRA_USER> -p <CASSANDRA_PASSWORD> -f emoney-storage.json -D

Reviewer 2: `-f` is not required for C*.

Author: Thank you for your suggestion.
Document updated with the above suggestion.

$ ../../gradlew run --args="-mode storage -action pay -amount 100 -to merchant1 -from user1"
```

## Set up database schema with transaction

To apply transaction, we can just add a key `transaction` and value as `true` in Scalar DB scheme.

```json
{
  "emoney.account": {
    "transaction": true,
    "partition-key": [
      "id"
    ],
    "clustering-key": [],
    "columns": {
      "id": "TEXT",
      "balance": "INT"
    }
  }
}
```

Before reapplying the schema, please drop the existing namespace first by issuing the following. 

> * Download `scalar-schema-<vesrion>.jar` file from [scalardb release](https://github.com/scalar-labs/scalardb/releases)

Reviewer: ```suggestion
```

Author: Thank you for the suggestion.
Document updated with the above suggestion.
